Games/types/Mafia/items/Gun.js

```diff
@@ -15,7 +15,7 @@ module.exports = class Gun extends Item {
     this.meetings = {
       [this.baseMeetingName]: {
         actionName: "Shoot",
-        states: ["Day"],
+        states: ["Day", "Night"],
```

Comment:
mentioned in comment below, i thought about it and think we shouldn't even implemented this, don't see its value other than being matchy with em

Reply:
agreed